lib/connect.c

```diff
@@ -621,3 +621,13 @@ CURLcode Curl_conn_setup(struct Curl_easy *data,
     Curl_resolv_unlink(data, &data->state.dns[sockindex]);
   return result;
 }
+
+void Curl_conn_set_multiplex(struct connectdata *conn, bool multiplex)
+{
```

Comment:
This function is never called to disable multiplexing again. Maybe it could be an explicit "enable multiplex" function?

Reply:
Done!